### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by siddharthkp** [auth0/cosmos#34]
cosmos should give people components that are accessible by default.

A lot of the work for accessibility is dependant on the product's flow, so we don't control that but the building blocks need to be _correct_ first. (Example: Add `for` for labels by default)

Need more research here on the best way to test for accessibility and make it part of our process so that there are no regressions

Credit: @jennz0r for making me realise that our design system should have this from the start.

**Comment by landitus** [comment]
Yes, I will add a more focused one so we can grab our first bit at it
